### PR TITLE
Document the goto_stubs mapping <leader>s

### PR DESCRIPTION
**Pull request by astier**
#988

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -58,9 +58,10 @@ generators, there is broad support.
 Apart from that, jedi-vim supports the following commands
 
 - Completion ``<C-Space>``
-- Goto assignments ``<leader>g`` (typical goto function)
-- Goto definitions ``<leader>d`` (follow identifier as far as possible,
+- Goto assignment ``<leader>g`` (typical goto function)
+- Goto definition ``<leader>d`` (follow identifier as far as possible,
   includes imports and statements)
+- Goto stub ``<leader>s``
```
Comment (blueyed) [inline README.rst]:
```suggestion
- Goto (typing) stub ``<leader>s``
```
?

Comment (blueyed) [reply README.rst]: and/or add an explanation similar to the lines above.

File: doc/jedi-vim.txt [hunk 2]
```diff
@@ -25,10 +25,11 @@ Contents                                *jedi-vim-contents*
...
     5.4  Go to definition (deprecated)  |g:jedi#goto_definitions_command|
-    5.5. Show documentation             |g:jedi#documentation_command|
-    5.6. Rename variables               |g:jedi#rename_command|
-    5.7. Show name usages               |g:jedi#usages_command|
-    5.8. Open module by name            |:Pyimport|
+    5.5  Go to stub                     |g:jedi#goto_stubs_command|
+    5.6. Show documentation             |g:jedi#documentation_command|
+    5.7. Rename variables               |g:jedi#rename_command|
+    5.8. Show name usages               |g:jedi#usages_command|
+    5.9. Open module by name            |:Pyimport|
```
Comment (blueyed) [inline doc/jedi-vim.txt]: Replace 5.4 ("Go to definition (deprecated)") with the new one?  (less diff churn, and phasing out the deprecated thing).

File: README.rst (hunk 1 above)
Comment (astier) [reply README.rst]: I personally find those explanations kind of redundant.

File: doc/jedi-vim.txt (hunk 2 above)
Comment (astier) [reply doc/jedi-vim.txt]: I pushed something.

**Comment by blueyed** [comment]
Thanks!

**Pull request merged**